This file is part of MXE. See LICENSE.md for licensing information.

Contains ad hoc patches for cross building.

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jonas Kvinge <jonas@jkvinge.net>
Date: Sat, 6 Feb 2021 01:18:29 +0100
Subject: [PATCH 2/2] Add gst_wasapi_util_initialize_com in
 gst_wasapi_util_get_device_enumerator


diff --git a/sys/wasapi/gstwasapiutil.c b/sys/wasapi/gstwasapiutil.c
index 1111111..2222222 100644
--- a/sys/wasapi/gstwasapiutil.c
+++ b/sys/wasapi/gstwasapiutil.c
@@ -348,6 +348,8 @@ gst_wasapi_util_get_device_enumerator (GstObject * self)
   HRESULT hr;
   IMMDeviceEnumerator *enumerator = NULL;
 
+  gst_wasapi_util_initialize_com();
+
   hr = CoCreateInstance (&CLSID_MMDeviceEnumerator, NULL, CLSCTX_ALL,
       &IID_IMMDeviceEnumerator, (void **) &enumerator);
   HR_FAILED_RET (hr, CoCreateInstance (MMDeviceEnumerator), NULL);
